core/assets/jsons/global.json

```diff
@@ -0,0 +1,68 @@
+{
+  "player":  {
+    "radius":      	0.4,
+    "bodytype":    	"dynamic",
+    "density":        1.0,
+    "friction":       0.0,
+    "restitution":    0.0,
+    "force": 		 80.0,
+    "damping": 		 10.0,
+    "maxspeed": 	  5.0,
+    "startframe":       0,
+    "walklimit":		4,
+    "flarecount":       5,
+    "sprintlightrad":  10,
+    "minlightradius":   2,
+    "texture":     	"player",
+    "debugcolor":  	"white",
+    "debugopacity":	192
+  },
+  "enemies": {
+    "typeA":  {
+      "radius": 0.4,
+      "bodytype": "dynamic",
+      "density": 1.0,
+      "friction": 0.0,
+      "restitution": 0.0,
+      "force": 80.0,
+      "damping": 10.0,
+      "maxspeed": 5.0,
+      "startframe": 0,
+      "walklimit": 4,
+      "texture": "enemy",
+      "debugcolor": "white",
+      "debugopacity": 192
+    }
+  },
+  "exit": {
+    "size":           [ 1.25, 1.25],
+    "bodytype":    	  "static",
+    "density":  	  0.0,
+    "friction":       0.0,
+    "restitution":    0.0,
+    "texture":       "goal",
+    "debugcolor":    "yellow",
+    "debugopacity":   200
+  },
+  "flare" : {
+    "radius":      	  0.1,
+    "bodytype":    	  "dynamic",
+    "density":        1.0,
+    "friction":       0.0,
+    "restitution":    0.0,
+    "texture":        "fire",
+    "initialforce":   3.0,
+    "damping": 		 0.0,
+    "startframe":       0
+  },
+  "wall": {
```

Comment:
Why are walls both in the global json and the level json/why is the information overlapping between the two? Is it the global one is a default wall and the level one is for specialization only?

Reply:
There isn't any overlapping information except texture. I put all information that is constant for walls in the global json, and put all dynamic data (size, pos, and texture) in the level json. 

I did put texture also in the global json thinking that could be the default texture if one is not specified in the level json, but I'm not committed to that and may room it (also the code doesn't handle that case yet.

Reply:
The default texture is now handled in WallModel (ie uses default if other is not specified).

Reply:
The "pad" attribute is overlapping in some walls (the second half of the walls in the level json), and it is the same value from the global. That's what I also meant in the "overlapping" aspect (in addition to texture).

Reply:
Oh whoops, yeah I can fix that